docs/pages/store.md

```diff
@@ -41,21 +46,17 @@ use Patchlevel\EventSourcing\Store\DoctrineDbalStore;
 $store = new DoctrineDbalStore(
     $connection,
     DefaultEventSerializer::createFromPaths(['src/Event']),
-    null,
-    [/** options */],
 );
 ```
 Following options are available in `DoctrineDbalStore`:
 
-| Option            | Type             | Default    | Description                                  |
-|-------------------|------------------|------------|----------------------------------------------|
-| table_name        | string           | eventstore | The name of the table in the database        |
-| aggregate_id_type | "uuid"|"string" | uuid       | The type of the `aggregate_id` column        |
-| locking           | bool             | true       | If the store should use locking for writing  |
-| lock_id           | int              | 133742     | The id of the lock                           |
-| lock_timeout      | int              | -1         | The timeout of the lock. -1 means no timeout |
-
-## Schema
+| Option            | Type            | Default    | Description                                  |
+|-------------------|-----------------|------------|----------------------------------------------|
+| table_name        | string          | eventstore | The name of the table in the database        |
+| aggregate_id_type | "uuid"/"string" | uuid       | The type of the `aggregate_id` column        |
+| locking           | bool            | true       | If the store should use locking for writing  |
+| lock_id           | int             | 133742     | The id of the lock                           |
+| lock_timeout      | int             | -1         | The timeout of the lock. -1 means no timeout |
```

Comment:
I created an issue to document more options for the bundle, see https://github.com/patchlevel/event-sourcing-bundle/issues/218